### PR TITLE
Add Skylink support to ENS resolver

### PR DESCRIPTION
**Pull request by peterjan**
Explanation:  
This PR extends the current ENS to IPFS resolver with the capability of resolving content hashes to skylinks. ENS added Skynet has been supported by ENS for a while now (cfr. https://github.com/ensdomains/content-hash/pull/7) and we would like ENS domains to be able to resolve to skylinks through MetaMask's listener.

Example: 
https://siasky.net/fAD570XqL8xQGJ3bxlS7RncsGes7ViGxcR5zYf5WXIuRNw

Manual testing steps:  
  - build the project
  - load extension in FireFox
  - type `skynetlabs.eth/` and hit enter in the address bar
  - verify it redirects to https://20062i9hn2944pcu789tdidf71i1s34hl1t2c4ncq7stdbc5i11a888.siasky.net/

### CODE REVIEW

**Comment by peterjan** [comment]
I have read the CLA Document and I hereby sign the CLA

File: app/scripts/lib/ens-ipfs/resolver.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import namehash from 'eth-ens-namehash';
 import Eth from 'ethjs-query';
 import EthContract from 'ethjs-contract';
-import contentHash from 'content-hash';
+import contentHash from '@ensdomains/content-hash';
```
Comment (makoto) [inline app/scripts/lib/ens-ipfs/resolver.js:4]: Hi. Makoto from ENS team. 
FYI,`content-hash` is no longer supported by the original author so we forked and took over the maintenance, so please use '@ensdomains/content-hash' as in this PR

**Comment by peterjan** [comment]
cc @ryanml is this something you would be able and willing to review? If not, could you perhaps refer us to someone that is? We would like to have some eyes on this PR  so we can potentially iterate on it and get it merged.

**Comment by peterjan** [comment]
@brad-decker sorry to tag you like this, but this PR is becoming a bit stale, is this something you would be able to review? If not perhaps point me in the direction of someone that is?

**Comment by brad-decker** [comment]
@peterjan looks good to me, other than the 'dist.zip' file -- could you remove that?

**Comment by peterjan** [comment]
> @peterjan looks good to me, other than the 'dist.zip' file -- could you remove that?

@brad-decker woops sorry! that snuck in, removed. Thank you for taking a look at it!

**Review by brad-decker — DISMISSED** [review]

**Comment by brad-decker** [comment]
@adonesky1 could you do the honors of providing the second required review and merge this?

**Comment by adonesky1** [comment]
@peterjan sorry I took too long on this. Do you mind rebasing and resolving conflicts?

**Comment by peterjan** [comment]
@adonesky1 @brad-decker merged upstream and resolved conflicts - thanks for taking the time guys!

**Review by brad-decker — APPROVED** [review]

**Review by adonesky1 — APPROVED** [review]

**Pull request merged**